src/qt/bitcoingui.cpp

```diff
@@ -962,6 +968,13 @@ void BitcoinGUI::showHelpMessageClicked()
     helpMessageDialog->show();
 }
 
+void BitcoinGUI::bootstrapClicked()
+{
+    auto bootdlg = new Bootstrapdlg(this);
```

Comment:
Can we call that dialog like the others ?

Reply:
fixing will resubmit pull request